app/_landing_pages/konnect-platform/authentication.yaml

```
@@ -27,13 +27,15 @@ rows:
             blocks:
               - type: text
                 text: |
-                  The default authentication option in {{site.konnect_short_name}} is basic authentication, where users can log in with a username and password.
+                  The default authentication option in {{site.konnect_short_name}} is basic authentication, where users can log in with a username and password. Users can also setup passkeys when they log into {{site.konnect_short_name}} using username and password.
                   
                   Users can also register for and sign in to {{site.konnect_short_name}} accounts using their social identity from the following providers:
                   * Google
                   * GitHub
                   * Microsoft
 
+                  Multi-Factor Authentication (MFA) is required for all users using built-in authentication or social login. Users will be prompted to set up their MFA when they login to Konnect for the first time. Please reach out to Kong Support to reset your MFA.
```

nits(non-blocking): login -> log in, their MFA -> MFA

```suggestion
                  Multi-Factor Authentication (MFA) is required for all users using built-in authentication or social login. Users will be prompted to set up MFA when they log in to Konnect for the first time. Please reach out to Kong Support to reset your MFA.
```